src/SingletonState.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Thunk\Verbs;
+
+use BadMethodCallException;
+use RuntimeException;
+use Thunk\Verbs\Lifecycle\StateManager;
+use Thunk\Verbs\Support\StateCollection;
+
+abstract class SingletonState extends State
+{
+    public static function make(...$args): static
+    {
+        throw new BadMethodCallException('To use singleton states, please call ::singleton() rather than ::make()');
```

Comment:
TBH I think this should all just defer to `static::singleton()`

Reply:
My thinking here was that I wanted the be sure that the calling code knew that it's a singleton. I could be talked out of this, though.